doc/CONTRIBUTING.md

```diff
@@ -146,12 +140,10 @@ Other notable changes in our patch include:
 
 As the web portion of VS Code matures, we'll be able to shrink and possibly
 eliminate our patch. In the meantime, upgrading the VS Code version requires
-us to ensure that the patch is applied and works as intended. In the future,
+us to ensure that our changes are applied and work as intended. In the future,
 we'd like to run VS Code unit tests against our builds to ensure that features
 work as expected.
```

Comment:
And here

Reply:
I think this makes sense as-is. Although, maybe `our changes are still applied`? Since they'll already be applied, the trick when updating will be to make sure they're still there and still work.

I think we might want to change the `eliminate our patch` part to something like `eliminate our modifications`.

Reply:
Updated!